src/Orchard.Cms.Web/Modules/Orchard.Setup/Views/Setup/Index.cshtml

```diff
@@ -106,6 +107,7 @@
                 <option value="@provider.Value" data-connection-string="@provider.HasConnectionString">@provider.Name</option>
                 }
             </select>
+            <span asp-validation-for="DatabaseProvider" class="text-danger"></span>
```

Comment:
We probably can, however I'm not sure. Ideally, the error message would span the full col-12, but if I add a wrapping `div`, that div will obviously be rendered regardless of there being an error or not. Any suggestions?